src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/mobile/perform_interaction_request.h

```diff
@@ -93,6 +97,32 @@ class PerformInteractionRequest
    */
   virtual void onTimeOut();
 
+  /**
+   * @brief Prepare result code for sending to mobile application
+   * @param first contains result_code from HMI response and
+   * interface that returns response
+   * @param second contains result_code from HMI response and
+   * interface that returns response.
+   * @return resulting code for sending to mobile application.
+   */
+  virtual mobile_apis::Result::eType PrepareResultCodeForResponse(
```

Comment:
@YarikMamykin Are you sure that it should be `virtual`?

Reply:
@AByzhynar fixed in https://github.com/smartdevicelink/sdl_core/pull/3122/commits/0190836450ab1d7e130bbb8e32315eb3325a9a68